io.github.OkuBrowser.oku.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id" : "io.github.OkuBrowser.oku",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "oku",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--socket=cups",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
+        "--filesystem=host",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=xdg-run/gvfsd",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
this needs an exception PR https://docs.flathub.org/docs/for-app-authors/linter#exceptions https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
Exception PR: https://github.com/flathub-infra/flatpak-builder-lint/pull/538